Dockerfile

```diff
@@ -1,60 +1,45 @@
-FROM ubuntu:18.04
-MAINTAINER Fabian Beuke <beuke@traum-ferienwohnungen.de>
+FROM ubuntu:20.04 
 
 RUN apt-get update &&                          \
     apt-get install -y --no-install-recommends \
+    add-apt-key                                \
     fontconfig                                 \
-    libcurl3                                   \
+    curl				                       \
```

Comment:
nit: out of line \

Reply:
Took it verbatim from the commit on the other fork. Serves me right.